pandas/core/apply.py

```diff
@@ -504,7 +505,10 @@ def apply_multiple(self) -> FrameOrSeriesUnion:
         return self.obj.aggregate(self.f, self.axis, *self.args, **self.kwargs)
 
     def normalize_dictlike_arg(
-        self, how: str, obj: FrameOrSeriesUnion, func: AggFuncTypeDict
+        self,
+        how: Literal['apply', 'agg', 'transform'],
```

Comment:
the linter is going to want double-quotes instead of single-quotes

Reply:
Updated this for all examples in the latest commits - thanks!